test/specs/test.js

```diff
@@ -1,8 +1,19 @@
-const MainPage = require('../pageobjects/mainPage');
+import mainPage from '../pageobjects/mainPage.js';
 
 describe('AccuWeather Test Suite', () => {
-    before(async () => {
-        await MainPage.open();
-        await MainPage.consentDataUsage();
+    it('consenting to data usage and allow cookie',async () => {
```

Comment:
You don't need to create 3 describe and 3 it 
Create just one and put you function in it

Reply:
Resolved